packages/search/src/search.stories.jsx

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { action } from '@storybook/addon-actions';
```

Comment:
ESLint probably complains here because the dependency is not declared in package's `devDependencies`? Adding it there should remove the need for disabling warnings.